20_classes_lab/01.py

```diff
@@ -1,6 +1,32 @@
-"""
-The following code assumes a class
-named Summer exists. 
-Complete the class so correct result is printed
-"""
-
+"""
+Fill the code to print the result
+"""
+
+class Summer(object):
+    def __init__(self):
+        self._temp = []
+
+    def add(self, *args):
+        for x in args:
+            if (type(x) is int): self._temp.append(x)
+
+    def total(self):
+        sum = 0

```

Comment:
1. Keeping just the sum as member is more efficient, since you don't need to calculate sum all over again every time `total` is called

2. `sum` is a bad variable name since it's already a keyword in python